JSDemos/Demos/PivotGrid/StandaloneFieldChooser/description.md

```diff
@@ -1 +1,14 @@
-The standalone field chooser is&nbsp;the&nbsp;PivotGridFieldChooser component that shares the data source with the PivotGrid and has three different layouts. Active layout changes can be&nbsp;applied to&nbsp;the PivotGrid in&nbsp;instant or&nbsp;on-demand mode.
\ No newline at end of file
+Along with a field chooser [integrated into the PivotGrid](https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/IntegratedFieldChooser/), DevExtreme includes a standalone [PivotGridFieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/) component. Unlike its integrated counterpart, the standalone field chooser can be displayed on a page permanently.
+
+Follow the steps below to connect the PivotGridFieldChooser with the PivotGrid:
+
+1. **Bind both components to the same data source**         
+Assign the same [PivotGridDataSource](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/) instance to the [dataSource](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#dataSource) property of both components.
+
+1. *(Optional)* **Disable the integrated field chooser**            
+Both field choosers can work simultaneously, but if you want to disable the integrated version, set the PivotGrid's [fieldChooser](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldChooser/).[enabled](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/fieldChooser/#enabled) property to **false**.
+
+1. *(Optional)* **Apply field changes on demand**           
+When users move fields within the field chooser, the changes are applied to the connected PivotGrid instantly. To change this behavior, you can set the [applyChangesMode](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#applyChangesMode) to *"onDemand"*. In this mode, the changes are applied/canceled only when you call the [applyChanges()](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Methods/#applyChanges)/[cancelChanges()](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Methods/#cancelChanges) method or set a new [state](/Documentation/ApiReference/UI_Components/dxPivotGridFieldChooser/Configuration/#state). You can add a UI for these methods. For example, in this demo, they are bound to the Apply and Cancel buttons. Select "onDemand" from the Apply Changes Mode drop-down menu to display these buttons.
```

Comment:
>You can add a UI for these methods.

Not sure that this sentence is valid. I would rephrase it as follows: 'You can call these methods from the UI' or something similar. 

However, if you are sure that this sentence is OK, just skip this comment.